Naming.java

```diff
@@ -0,0 +1,31 @@
+package az.alizeynalli.cleancode.naming;
+
+public class Naming {
+
+    public Date d;
+    public List<Testcase> find(User user){};
+    
+    public Account[] AccountList;
```

Comment:
// use Suffix List, if it is really a List
    //Avoid misinformation
    // Good
    public List<Account> AccountList;